### DISCUSSION TITLE
is there a way to make separate action classes?

### DISCUSSION THREAD

**Discussion by jdion84** [filamentphp/filament#9484] [Help] [answered]
i cant find anything in the docs

i would like to put my actions in their own classes cause they make the resource classes huge

**Comment by leandrocfe** [comment]
What is the issue?

**Reply by jdion84** [reply]
its not a big issue, was just curious more than anything

**Reply by leandrocfe** [reply]
You can create custom classes to organize your code.

**Comment by sinnbeck** [comment, marked as answer]
Just add your own class that extends the Action class from filament. Check one of the built in from filament to see how its done

https://github.com/filamentphp/actions/blob/3.x/src/ViewAction.php
